website/docs/d/subscriptions.html.markdown

```diff
@@ -36,6 +36,7 @@ output "first_available_subscription_display_name" {
 
 The `subscription` block contains:
 
+* `resource_manager_id` - The Resource Manager ID of this subscription.
```

Comment:
can we make this just `id` to match other resources:

```suggestion
* `id` - The ID of this Subscription.
```

Reply:
I was a bit confused if I should use `id` or `resource_manager_id` ;) It fits "better" and it compatible with azurerm_subscription, but I've seen `resource_manager_id` in a lot of different places too. Seems I've picked wrong.
I'll change it soon.

Reply:
no worries - we use `resource_manager_id` only where a secondary is present (e.g. data plane id) - otherwise this should be `id`